packages/react-components/src/components/attribution/index.tsx

```diff
@@ -37,6 +37,7 @@ const Attribution = ({ gif, className }: Props) => {
             className={cx(containerCss, Attribution.className, className)}
             onClick={e => {
                 e.preventDefault()
+                callback?.()
```

Comment:
@nicksantan maybe you can name this `onClick`, also should it still open the window if `onClick` is defined? I feel like it shouldn't

Reply:
Thanks for the quick thoughts @giannif -- I was having that same thought. For my use case, I'd like to fire a GA handler and then proceed with the normal onClick behavior... so I figured maybe the callback should fire in addition rather than in place of the window open. 

I could also go your way and have it fire instead of the window.open and then just do that manually in the web app.

Reply:
@nicksantan yeah I think if you define the `onClick` handler, you might want to override the behavior, and there'd be no way to do so.

 It'd be easy to reimplement the default behavior if perhaps the callback was invoked with the gif: `onClick(gif: IGif)`

Reply:
👍 Thanks! Changes to be made shortly -

Reply:
@giannif one thing that occurred to me - will there be conflicts with the native onClick handler if the prop is also called onClick?